parsely/parselyandroid/ParselyTracker.java

```diff
@@ -152,55 +152,92 @@ public static ParselyTracker sharedInstance(String apikey, int flushInterval, St
+     * will enqueue events less frequently over time to save data.
+     *
+     * @return The base engagement tracking interval.
+     */
     public double getEngagementInterval() {
         return DEFAULT_ENGAGEMENT_INTERVAL_MILLIS;
     }
 
+    /*! \brief Returns whether the engagement tracker is running.
+     *
+     * @return Whether the engagement tracker is running.
+     */
     public boolean engagementIsActive() {
         return this.engagementManager != null;
     }
 
+    /*! \brief Returns whether video tracking is active.
+     *
+     * @return Whether video tracking is active.
+     */
     public boolean videoIsActive() {
         return this.videoEngagementManager != null;
     }
 
+    /*! \brief Returns the interval at which the event queue is flushed to Parse.ly.
+     *
+     * @return The interval at which the event queue is flushed to Parse.ly.
+     */
+    public long getFlushInterval() {
+        return this.flushManager.getIntervalMillis() / 1000;
+    }
+
     /*! \brief Getter for this.isDebug
+     *
+     * @return Whether debug mode is active.
      */
     public boolean getDebug() {
         return isDebug;
     }
 
     /*! \brief Set a debug flag which will prevent data from being sent to Parse.ly
-
-        Use this flag when developing to prevent the SDK from actually sending requests
-        to Parse.ly servers.
-
-        @param debug Value to use for debug flag.
+     *
+     *  Use this flag when developing to prevent the SDK from actually sending requests
+     *  to Parse.ly servers. The value it would otherwise send is logged to the console.
+     *
+     *  @param debug Value to use for debug flag.
      */
     public void setDebug(boolean debug) {
         isDebug = debug;
         PLog("Debugging is now set to " + isDebug);
     }
 
-    /*! \brief Register a pageview event using a canonical URL
+    /*! \brief Register a pageview event using a URL and optional metadata.
      *
-     *  @param url The canonical URL of the article being tracked
-     *  (eg: "http://samplesite.com/some-old/article.html")
+     *  @param url         The canonical URL of the article being tracked
+     *                     (eg: "http://samplesite.com/some-old/article.html")
+     *  @param urlMetadata Optional metadata for the URL -- not used in most cases. Only needed
+     *                     when `url` isn't accessible over the Internet (i.e. app-only
+     *                     content). Do not use for URLs that Parse.ly would normally crawl.
      */
     public void trackURL(String url, ParselyMetadata urlMetadata) {
         this.enqueueEvent(this.buildEvent(url, "pageview", urlMetadata));
     }
 
+    /*! \brief Start engaged time tracking for the given URL.
+     *
+     * This starts a timer which will send events to Parse.ly on a regular basis
+     * to capture engaged time for this URL. This URL should match the value passed
```

Comment:
Imprecise - which call to `trackURL` does a call to this function need to match? Suggestion:
> The URL passed to this function should be a URL for which `trackURL` has been called.